docs/workflows/stories-for-multiple-components.md

```diff
@@ -61,9 +61,18 @@ One way we improve that situation is by pulling the rendered subcomponent out in
 
 Now that `children` is an arg, we can potentially reuse it in another story.
 
+However there are some caveats when using this approach that you should be aware of.
+
+The `children` `args` as any other arg needs to be JSON serializable. This means that you should:
+
+- avoid using empty values.
+- avoid using components that include third party libraries such as `emotion`.
```

Comment:
not sure we know enough to put this in the docs. i think emotion does work in general, but fails in some cases--not sure which ones yet.

Reply:
@shilman  as you said it we don't know much about it yet, and it will be a bit of "pickle" (pardon the bad pun) to hunt down each and every use case, for now we could probably  frame it as :

```md
The `children` `args` as any other arg needs to be JSON serializable. This means that you should:
-  avoid using empty values
- use caution with components that include third party libraries
```

Let me know if you're ok with the changes and i'll commit them and we get it in the docs